### ISSUE TITLE
Need more detail about format for submissions

### ISSUE THREAD

**Issue by otherjoel** [Racket-Cookbooks/Scribble-cookbook#1]
I have something to submit, but there’s no guidance about how to format entries, and no examples yet to follow.

When people come to look at the Scribble Cookbook, what do you envision that they should see? Are they seeing all the “recipes” as code and screenshots directly in the README?

Ideally, shouldn’t the cookbook itself be a Scribble document? Perhaps also published on the package server / rendered at `docs.racket-lang.org`? This makes submissions a little more trouble to create and format, but also possibly easier to use.

If the latter is something you are open to, I could do a PR setting up the Scribble doc, structure, instructions etc.

**Comment by spdegabrielle** [comment]
Yes please!

**Comment by bennn** [comment]
Ideally yes, the cookbook should be a Scribble document. But how that would work is tough question since a scribble program might output any number of things ... anything from a website to a PDF document to a nice string (for other documents to incorporate).

Building those outputs might require some setup, too.

Still, any infrastructure would be better than nothing!

The easy starting point is to put each new example in its own directory with a README containing build instructions & sample output.

A shared README with screenshots is a great intermediate step.

**Comment by spdegabrielle** [comment]
Let’s stick with `scribble/manual`and host at docs.racket-lang.org. The should keep the barriers to contribution low.

For the quickscript @metaxal came up with a little script to generate the markdown manually - this is then committed to the repo. I’m sure the same approach will work and we can link to other renderings.

**Comment by otherjoel** [comment]
The initial idea I had for a contribution involves not just scribble but an auxiliary CSS file (ideally a TeX file too) for style additions. So having a way to bundle all the associated files for an example is kind of a minimal requirement (which of course the sub folder + Readme approach would cover).

**Comment by spdegabrielle** [comment]
> The initial idea I had for a contribution involves not just scribble but an auxiliary CSS file (ideally a TeX file too) for style additions. So having a way to bundle all the associated files for an example is kind of a minimal requirement (which of course the sub folder + Readme approach would cover).

That is awesome. 

It would be nice to be able to embed racket code like slideshow, perhaps using Racketscript.

**Comment by otherjoel** [comment]
I'll probably submit when there's more of a skeleton of a thing to add on to. I haven’t had time to create everything from scratch in order to make a submission.

**Comment by bennn** [comment]
I finally added quick instructions to the README to encourage submissions in standalone top-level folders.

TLDR make your own directory & include steps to reproduce a small demo & we'll merge it ❤

**Comment by spdegabrielle** [comment]
thanks @bennn
